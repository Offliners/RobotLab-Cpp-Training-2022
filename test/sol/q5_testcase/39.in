8-1+19-1-8*6/15=/19/10+6-15-14=*12/18/2+15+13=-4-14-3=+18/20*13=+2-15/5%9=
