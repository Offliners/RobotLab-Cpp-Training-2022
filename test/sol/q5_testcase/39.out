181
182
175
183
